cv-data-service-library/src/main/java/com/trihydro/library/helpers/JsonToJavaConverter.java

```diff
@@ -242,7 +242,14 @@ else if (reverse != null)
         }
 
         if (regionDirectionNode != null) {
-            region.setDirection(mapper.treeToValue(regionDirectionNode, String.class));
+            StringBuilder directionBuilder = new StringBuilder();
+            Iterator<Map.Entry<String, JsonNode>> fields = regionDirectionNode.fields();
+            while (fields.hasNext()) {
+                Map.Entry<String, JsonNode> field = fields.next();
+                boolean directionBool= Boolean.parseBoolean(field.getValue().toString());
+                directionBuilder.append(directionBool ? "1" : "0");
+            }
+            region.setDirection(directionBuilder.toString());
```

Comment:
Question (blocking): This appears to be using the direction bitstring, but I believe Rick told me that we are moving away from that element in favor of a different direction-related element. Should we be doing this differently?

Reply:
You're correct here, we'll want to remove this in favor of "forward"

Reply:
Paige and I confirmed with Rick that either bitmask or directionality is valid, but not both together. We should update the conditionals to ensure only one is used at a time.